resources/rgbscripts/alternate.js

```diff
@@ -72,6 +72,9 @@ var testAlgo;
 
   var x = 0;
   var y = 0;
+  // current RGB assigned to the step
```

Comment:
Remove this?

Reply:
Yes, it is a leftover. Not sure why, I should have seen these things in my final review before opening the pull request. Thanks for double-checking my code. I'll clean that up.